Dịch chapter_preliminaries/linear-algebra_vn - Phần 08 (#428)
aivivn/d2l-vn: chapter_preliminaries/linear-algebra_vn.md

## Matrix-Vector Products
-->

## Tích giữa ma trận và vector

<!--
Now that we know how to calculate dot products, we can begin to understand *matrix-vector products*.
Recall the matrix $\mathbf{A} \in \mathbb{R}^{m \times n}$ and the vector $\mathbf{x} \in \mathbb{R}^n$ defined and visualized in :eqref:`eq_matrix_def` and :eqref:`eq_vec_def` respectively.
Let's start off by visualizing the matrix $\mathbf{A}$ in terms of its row vectors
-->

Giờ đây, khi đã biết cách tính toán tích vô hướng, chúng ta có thể bắt đầu hiểu *tích giữa ma trận và vector*.

> Bạn có thể xem lại cách ma trận $\mathbf{A} \in \mathbb{R}^{m \times n}$ và vector $\mathbf{x} \in \mathbb{R}^n$ được định nghĩa và hiện thị trong :eqref:`eq_matrix_def` and :eqref:`eq_vec_def`.

Reviewer: ```suggestion
Bạn có thể xem lại cách ma trận $\mathbf{A} \in \mathbb{R}^{m \times n}$ và vector $\mathbf{x} \in \mathbb{R}^n$ được định nghĩa và biểu diễn trong :eqref:`eq_matrix_def` và :eqref:`eq_vec_def`.
```

> Ta sẽ bắt đầu bằng việc hiện thị ma trận $\mathbf{A}$ qua các vector hàng của nó. 

Reviewer: ```suggestion
Ta sẽ bắt đầu bằng việc biểu diễn ma trận $\mathbf{A}$ qua các vector hàng của nó. 
```

\vdots \\
\mathbf{a}^\top_m \\
\end{bmatrix},$$

<!--
where each $\mathbf{a}^\top_{i} \in \mathbb{R}^n$ is a row vector representing the $i^\mathrm{th}$ row of the matrix $\mathbf{A}$.
The matrix-vector product $\mathbf{A}\mathbf{x}$ is simply a column vector of length $m$, whose $i^\mathrm{th}$ element is the dot product $\mathbf{a}^\top_i \mathbf{x}$:
-->

Mỗi $\mathbf{a}^\top_{i} \in \mathbb{R}^n$ là một vector hàng thể hiện hàng thứ $i$ của ma trận $\mathbf{A}$.

> Tích giữa ma trận và vector $\mathbf{A}\mathbf{x}$ đơn giản chỉ là một vector cột với chiều dài $m$, với phần tử thứ $i$ là kết quả của tích vô hướng $\mathbf{a}^\top_i \mathbf{x}$:

Reviewer: ```suggestion
Tích giữa ma trận và vector $\mathbf{A}\mathbf{x}$ đơn giản chỉ là một vector cột với chiều dài $m$, với phần tử thứ $i$ là kết quả của phép tích vô hướng $\mathbf{a}^\top_i \mathbf{x}$:
```

As we will see in subsequent chapters, we can also use matrix-vector products to describe the most intensive calculations required when computing each layer in a neural network given the values of the previous layer.
-->

> Chúng ta có thể nghĩ đến việc nhân một ma trận $\mathbf{A}\in \mathbb{R}^{m \times n}$ với một vector như một phép biến hình chiếu vector $\mathbb{R}^{n}$ thành $\mathbb{R}^{m}$.

Reviewer: ```suggestion
Chúng ta có thể nghĩ đến việc nhân một ma trận $\mathbf{A}\in \mathbb{R}^{m \times n}$ với một vector như một phép biến hình, chiếu vector từ không gian $\mathbb{R}^{n}$ thành $\mathbb{R}^{m}$.
```

> Những phép biến hình này trở nên rất hữu dụng. 

Reviewer: ```suggestion
Những phép biến hình này hóa ra lại trở nên rất hữu dụng. 
```

> Ví dụ, chúng ta có thể biểu thị phép xoay bởi tích với một ma trận vuông.

Reviewer: ```suggestion
Ví dụ, chúng ta có thể biểu diễn phép xoay là tích với một ma trận vuông.
```

> Bạn sẽ thấy ở những chương tiếp theo, chúng ta cũng có thể sử dụng tích giữa ma trận và vector để thực hiện hầu hết những tính toán cần thiết khi tính mỗi lớp trong một mạng nơ-ron dựa theo kết quả của lớp trước đó.

Reviewer: ```suggestion
Bạn sẽ thấy ở những chương tiếp theo, chúng ta cũng có thể sử dụng tích giữa ma trận và vector để thực hiện hầu hết những tính toán cần thiết khi tính các tầng trong một mạng nơ-ron dựa theo kết quả của tầng trước đó.
```

Mình dùng `tầng` cho `layer` để tránh đụng hàng với `class`
